composer.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "concrete5-community-store/community_store",
-  "type": "concrete5-package",
-  "require": {
-    "php": ">=5.6",
-    "composer/installers": ">=1.0.7",
-    "select2/select2": "3.5.4"
-  },
-  "minimum-stability": "dev",
-  "support": {
-    "source": "https://github.com/concrete5-community-store/community_store/src"
-  }
-
-}
+    "name": "concrete5-community-store/community_store",
+    "description": "An open, free and community developed eCommerce system for Concrete CMS",
+    "type": "concrete5-package",
+    "homepage": "https://github.com/concretecms-community-store/community_store",
+    "license": "MIT",
+    "require": {
+        "php": ">=5.6"
```

Comment:
Is PHP 5.6 the minimum PHP version?

Reply:
I think that would still be correct, but it probably makes sense to have it 7.4, to better align it with V8 requirements

Reply:
Isn't Community Store [compatible with concrete5 v8.5](https://github.com/concretecms-community-store/community_store/blob/v2.4.8.5/controller.php#L22)?
If so, the minimum PHP version is 5.5.
If Community Store is only compatible with ConcreteCMS 9+, the minimum PHP version is [7.3](https://github.com/concretecms/concretecms/blob/9.0.0/concrete/composer.json#L28).

Reply:
It should be compatible with version 8 in general.
I think I set it to 5.6, as there was some code in there that wasn't playing nicely with 5.5.